Server/Components/Pawn/Scripting/Types.hpp

```diff
@@ -23,7 +23,7 @@ template <>
 class ParamCast<PawnScript&>
 {
 public:
-	ParamCast(AMX* amx, cell* params, int idx)
+	ParamCast(AMX* amx, cell* params, int idx, int& error = ParamCastErrorCode_None)
```

Comment:
int& shouldn't have a default value

Reply:
Entirely changed how it works now, no more ref args, handled internally in each ParamCast

Reply:
There were 3 error values before but now it's just a bool, what happened to the third one?

Reply:
Not needed anymore because the old ones weren't used, only there for future if it would be ever needed but since now are error handlings actually happen in every ParamCast class, user/dev can make their own version of it (like printing something in case of failure), and return true at the end in ParamCast::Error().

We (pawn-native) only want to know if it failed or not, just to stop script native execution